web/checks_api.go

```diff
@@ -60,6 +61,11 @@ func ApiChecksCatalogHandler(s services.ChecksService) gin.HandlerFunc {
 			g := JSONChecksGroup{Group: group.Group, Checks: group.Checks}
 			groupedCatalog = append(groupedCatalog, &g)
 		}
+
+		sort.SliceStable(groupedCatalog, func(i, j int) bool {
```

Comment:
Hey @dottorblaster ,
This is not the correct way of ordering, it is not alphabetical hehe
The order of the groups is based on the name of the checks. So th
This was already codified this way here: https://github.com/trento-project/trento/blob/main/web/models/check.go#L91

The correct fix would be to add the ordering in line 59 like this:
```
for _, group := range checkGroups.OrderByName() {
```